exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb

```diff
@@ -36,20 +36,22 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #105.        Creates a UI form for editing the Exasol Docker-DB configuration.
</pre><p>Copy &amp; Paste error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Well spotted! Thanks.